help/destinations/catalog/crm/microsoft-dynamics-365.md

```diff
@@ -0,0 +1,218 @@
+
+Read [Activate profiles and segments to streaming segment export destinations](../../ui/activate/activate-segment-streaming-destinations.md) for instructions on activating audience segments to this destination.
+
+### Mapping considerations and example {#mapping-considerations-example}
+
+To correctly send your audience data from Adobe Experience Platform to the [!DNL Dynamics 365] destination, you need to go through the field mapping step. Mapping consists of creating a link between your Experience Data Model (XDM) schema fields in your Platform account and their corresponding equivalents from the target destination. To correctly map your XDM fields to the [!DNL Dynamics 365] destination fields, follow these steps:
+
+1. In the [UICONTROL Mapping] step, click **[!UICONTROL Add new mapping]**, you will see a new mapping row on the screen.
+![Platform UI screenshot example for Add new mapping.](../../assets/catalog/crm/microsoft-dynamics-365/add-new-mapping.png)
+
+1. In the [!UICONTROL Select source field] window, when selecting the source field choose the **[!UICONTROL Select identity namespace]** category and select `contactId`.
+![Platform UI screenshot example for Source mapping.](../../assets/catalog/crm/microsoft-dynamics-365/source-mapping.png)
+
+1. In the [!UICONTROL Select target field] window, select the type of target field that you want to map your source field to.
+    * **[!UICONTROL Select identity namespace]**: select this option to map your source field to an identity namespace from the list.
+    ![Platform UI screenshot showing Target mapping for contactId.](../../assets/catalog/crm/microsoft-dynamics-365/target-mapping-contactid.png)
+
+    * Add the following mapping between your XDM profile schema and your [!DNL Dynamics 365] instance:
+        |XDM Profile Schema|[!DNL Dynamics 365] Instance| Mandatory|
+        |---|---|---|
+        |`contactId`|`contactId`| Yes |
+
+    * **[!UICONTROL Select custom attributes]**: select this option to map your source field to a custom attribute that you define in the [!UICONTROL Attribute name] field. Refer to [[!DNL Dynamics 365] documentation](https://docs.microsoft.com/en-us/dynamics365/customerengagement/on-premises/developer/entities/contact?view=op-9-1#entity-properties) for a comprehensive list of supported attributes.
+    ![Platform UI screenshot showing Target mapping for LastName.](../../assets/catalog/crm/microsoft-dynamics-365/target-mapping-lastname.png)
+
+        > [!IMPORTANT]
+        >
+        > When mapping to a [!DNL Dynamics 365] [date or timestamp](https://docs.microsoft.com/en-us/power-apps/developer/data-platform/webapi/reference/timestampdatemapping?view=dataverse-latest) attribute, the corresponding attribute value in the dataset should not be empty. During execution if the value is empty you will encounter errors.
```

Comment:
Line 146: "the corresponding attribute value in the dataset should not be empty."

Which dataset? Whici attribute value? The one in Dynamics or the one in Platform? Please clarify this note with engineering, as it sounds confusing in this state. Reading it now, I don't really understand what it refers to.

Reply:
Have rephrased.